src/pattern-library/components/patterns/prototype/LMSContentSelectionPage.tsx

```diff
@@ -0,0 +1,758 @@
+            {contentType}
+          </span>
+        </div>
+      </div>
+    </ButtonBase>
+  );
+}
+
+export default function LMSContentSelectionPage() {
+  return (
+    <Library.Page
+      title="LMS content selection UI"
+      intro={
+        <p>
+          UI problem: There are too many heavy, identical, wordy buttons on the
+          LMS content-selection screen, and they are starting to exceed
+          available space.
+        </p>
+      }
+    >
+      <Library.Section>
+        <Library.Pattern title="Current UI">
+          <Library.Example>
+            <Library.Demo>
+              <Card>
+                <CardHeader variant="secondary" title="Assignment details" />
+                <CardContent size="lg" classes="leading-none">
+                  <div className="grid grid-cols-[auto_1fr] gap-x-6 gap-y-3">
+                    <div className="text-end leading-none text-[14px]">
+                      <span className="font-medium text-slate-7">
+                        Assignment content
+                      </span>
+                    </div>
+                    <div className="space-y-3 text-[13px]">
+                      <p>
+                        You can select content for your assignment from one of
+                        the following sources:
+                      </p>
+                      <div className="flex flex-row p-y-2">
+                        <div className="flex flex-col space-y-1">
+                          <Button variant="primary">
+                            Enter URL of web page or PDF
+                          </Button>
+                          <Button variant="primary">
+                            Select PDF from Canvas
+                          </Button>
+                          <Button variant="primary">
+                            Select PDF from Google Drive
+                          </Button>
+                          <Button variant="primary">
+                            Select JSTOR article
+                          </Button>
+                          <Button variant="primary">
+                            Select PDF from OneDrive
+                          </Button>
+                          <Button variant="primary">
+                            Select book from VitalSource
+                          </Button>
+                        </div>
+                        <div className="grow" />
+                      </div>
+                    </div>
+                  </div>
+                </CardContent>
+              </Card>
+            </Library.Demo>
+          </Library.Example>
+        </Library.Pattern>
+
+        <Library.Pattern title="Iterative: Round 1">
+          <Library.Example title="Use grid layout">
+            <p>
+              It is quick to put the buttons in a two-column grid layout.
+              However, because of their heavy, dark color, one ends up with an
+              unpleasant optical illusion between button corners.
```

Comment:
I see a light grey "diamond" illusion appear in the white space where the four button corners meet. Is that what you mean by optical illusion?

Reply:
Yes. Is there clearer wording I could use?